src/lightning/pytorch/core/optimizer.py

```diff
@@ -334,6 +335,12 @@ def _validate_scheduler_api(lr_scheduler_configs: List[LRSchedulerConfig], model
 
 
 def _validate_multiple_optimizers_support(optimizers: List[Optimizer], model: "pl.LightningModule") -> None:
+    if "optimizer_idx" in inspect.signature(model.training_step).parameters:
```

Comment:
We have `is_param_in_hook_signature` for these checks ([example](https://github.com/Lightning-AI/lightning/blob/a23bae39c4a1ea321275d389752b31aac9dfe9eb/src/lightning/pytorch/trainer/configuration_validator.py#L154)).

Reply:
That's awesome, I updated that.